scrapy_splash/cache.py

```diff
@@ -7,13 +7,18 @@
 """
 from __future__ import absolute_import
 import os
+from warnings import warn
 
 from scrapy.extensions.httpcache import FilesystemCacheStorage
 
 from .dupefilter import splash_request_fingerprint
 
 
 class SplashAwareFSCacheStorage(FilesystemCacheStorage):
+    def __init__(self, settings):
+        warn("Use the REQUEST_FINGERPRINTER_CLASS Scrapy setting instead", DeprecationWarning, stacklevel=2)
```

Comment:
No strong opinion, but maybe we could be more verbose here and in similar deprecation warnings, e.g.

```suggestion
        warn(
            (
                "scrapy-splash.SplashAwareFSCacheStorage is deprecated. Set "
                "the REQUEST_FINGERPRINTER_CLASS Scrapy setting to "
                "\"scrapy_splash.SplashRequestFingerprinter\" instead."
            ),
            DeprecationWarning, 
            stacklevel=2,
        )
```